templates/ansible/playbooks/openshift/install/address-space-controller/020-ConfigMap-address-space-definitions.yaml

```diff
@@ -849,7 +849,7 @@ data:
                 value: "INFER"
               - name: QDROUTERD_AUTO_MESH_SERVICE_NAME
                 value: "qdrouterd-headless-${INFRA_UUID}"
-              image: docker.io/enmasseproject/router:0.23.0
+              image: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/amq-interconnect/amq-interconnect-1.2-openshift:1.0-6
```

Comment:
I think this should be just 'amq-interconnect/amq-interconnect-1.2...' and not include internal red hat registry urls.